convert_lora_to_gguf.py

```diff
@@ -323,13 +350,15 @@ def __init__(self, *args, dir_lora_model: Path, lora_alpha: float, **kwargs):
                 self.dir_model_card = dir_lora_model
                 self.lora_alpha = float(lora_alpha)
 
+            def set_vocab(self):
+                pass
```

Comment:
I agree with not storing redundant metadata (like the whole tokenizer) from the base model in the LoRA adapter files. It wasn't used from there anyway.

Reply:
Yup thanks for the confirmation. Indeed, there is a second motivation to remove this. On the HF hub, GGUF files having complete metadata will be tagged as "llama.cpp model", which will be quite confused for end-user (i.e. some may attempt to load this adapter as a base model)